app/serializers/provider_serializer.rb

```diff
@@ -41,7 +41,7 @@ class ProviderSerializer < ActiveModel::Serializer
              # see generate_provider_contacts and return_application_alert_recipient for logic.
 
   attribute :contacts do
-    generate_provider_contacts + application_alert_recipient
+    generate_contacts
```

Comment:
Not sure it's worth having this as separate method at this point, unless it's used somewhere else?